src/prisma/migrations/20230718121709_extend_space_page_ops/migration.sql

```diff
@@ -0,0 +1,14 @@
+-- AlterEnum
+ALTER TYPE "PageOperations" ADD VALUE 'delete_attached_bounty';
```

Comment:
i don't know where the design doc is but i'm curious if this is supposed to be a separate admin setting.. couldn't we just rely on page permissions, and consider this bounty as a proeprty of the page?

Reply:
It's covering an edge case, where the user has deleteAnyBounty, but not deleteAnyPage

That way, they could delete a bounty from a card page, without deleting the page. Since we are soon going to have alot more intensive bounty attachements, felt like a good time to plug it in